gapis/api/templates/vulkan_gfx_api_extras.tmpl

```diff
@@ -221,6 +228,19 @@ bool Vulkan::replayCreateVkDeviceImpl(Stack* stack, size_val physicalDevice,
       return false;
     }
   }
+¶
+  bool Vulkan::replayDestroyVkInstance(Stack* stack) {
+    auto pAllocator = stack->pop<VkAllocationCallbacks const*>();
+    auto instance = static_cast<size_val>(stack->pop<size_val>());
+    // Leak this first instance. This is to work around a driver bug.
```

Comment:
Do we need to check if the stack is valid or not here?

Reply:
Thanks. We should. I will update.